packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -147,7 +151,11 @@
     as
     <code class="dummy-code">ToggleIcon</code>):</p>
   <dl class="dummy-component-props" aria-labelledby="component-api-toggle-icon">
-    <dt>text <code>string</code></dt>
+    <dt>
+      text
+      <code>string</code>
+      <strong class="required">required</strong>
```

Comment:
[issue] there are other props in the different dropdown sub-components that probably need a `required` tag